packages/grafana-ui/src/components/Table/TableNG/hooks.test.ts

```diff
@@ -418,27 +418,14 @@ describe('TableNG hooks', () => {
       const { result } = renderHook(() => {
         const data = createDataFrame({ fields: [numericField, numericField2] });
         cacheFieldDisplayNames([data]);
-        return useFooterCalcs(rows, data, data.fields, {
+        return useFooterCalcs(rows, data.fields, {
           enabled: true,
           footerOptions: { show: true, reducer: ['sum'], fields: [] },
         });
       });
 
       expect(result.current).toEqual(['6', '13']);
     });
-
-    it('should use fields and not data.fields as the loop to render over', () => {
-      const { result } = renderHook(() => {
-        const data = createDataFrame({ fields: [numericField, numericField2] });
-        cacheFieldDisplayNames([data]);
-        return useFooterCalcs(rows, data, data.fields.slice(0, 1), {
-          enabled: true,
-          footerOptions: { show: true, reducer: ['sum'], fields: [] },
-        });
-      });
-
-      expect(result.current).toEqual(['6']);
-    });
```

Comment:
this test is no longer necessary, as it was just an assurance that we didn't accidentally use `data.fields` instead of `fields`, but we no longer pass `data`.